backend/src/middleware/errorHandler.ts

```
@@ -0,0 +1,79 @@
+    error: {
+      message,
+      code,
+      status
+    }
+  };
+};
+
+/**
+ * Async handler to catch errors in async route handlers
+ */
+export const asyncHandler = (fn: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)